ExportHtml.py

```diff
@@ -152,19 +159,21 @@
     '</tr>'
 )
 
+CODE_LINE = (
+    '<span id="L_%(table)d_%(line_id)d" class="code_text code_gutter" style="color: %(color)s;">' +
+    '%(line)s</span><span class="code_line"><span id="C_%(table)d_%(code_id)d">%(code)s</span></span>\n'
```

Comment:
I think this is where it can be collapsed.

Reply:
I think code comes back as it's own span, but I think it's possible to collapse it. I'll have to check what the HTML scripts do with the targeted elements etc. But looks promising.